lib/applicants/validation.ts

```diff
@@ -23,11 +23,7 @@ export const requestPermitHolderInformationSchema = object({
       then: mixed<Gender>().oneOf(Object.values(Gender)).required('Please select a gender'),
       otherwise: mixed<Gender>().oneOf(Object.values(Gender)).optional(),
     }),
-  otherGender: string().when('type', {
-    is: 'NEW',
-    then: string().nullable().default(null),
```

Comment:
I'm curious why this didn't work? When the request type is NEW, isn't the logic exactly the same as the new version?

Reply:
I checked that the type is indeed 'NEW' for new applicants, so unless I am misinterpreting how the `yup` API works, I'm not sure where the issue is.